tests/files/你好世界😊.txt

```diff
@@ -0,0 +1 @@
+This file has unicode in its filename!
```

Comment:
I don't think we should have weird characters in the filename since I think it screws up Windows installations I believe.  We can put it in a zip file though.

Reply:
I'll try this on my windows machine and report if it breaks it. I believe it's spaces in filenames that really break windows, chinese and emoji should be fine.